runtime-light/stdlib/system/system-functions.h

```diff
@@ -60,6 +79,8 @@ inline string f$php_uname(const string &mode = string{1, 'a'}) noexcept {
   }
 }
 
+string f$php_sapi_name() noexcept;
```

Comment:
if you address previous comments, there will be no need in moving this implementation into the `cpp` file.